tests/scripts/thread-cert/config.py

```diff
@@ -64,8 +64,9 @@
 DOMAIN_PREFIX_ALTER = 'fd00:7d04:7d04:7d04::/64'
 
 PORT_OFFSET = int(os.getenv('PORT_OFFSET', '0'))
-BACKBONE_PREFIX = f'{0x9100 + PORT_OFFSET:04x}::/64'
-BACKBONE_PREFIX_REGEX_PATTERN = f'^{0x9100 + PORT_OFFSET:04x}:'
+BACKBONE_IPV6_ADDR_START_BASE = 0x9100
+BACKBONE_PREFIX = f'{BACKBONE_IPV6_ADDR_START_BASE + PORT_OFFSET:04x}::/64'
+BACKBONE_PREFIX_REGEX_PATTERN = f'^{BACKBONE_IPV6_ADDR_START_BASE + PORT_OFFSET:04x}:'
```

Comment:
```suggestion
BACKBONE_IPV6_ADDR_START_BASE = 0x9100
BACKBONE_IPV6_ADDR_START = BACKBONE_IPV6_ADDR_START_BASE + PORT_OFFSET
BACKBONE_PREFIX = f'{BACKBONE_IPV6_ADDR_START:04x}::/64'
BACKBONE_PREFIX_REGEX_PATTERN = f'^{BACKBONE_IPV6_ADDR_START:04x}:'
```
nit

Reply:
how about removing `BACKBONE_IPV6_ADDR_START_BASE `, directly make
```
BACKBONE_IPV6_ADDR_START = 0x9100  + PORT_OFFSET
```

Reply:
SGTM

Reply:
emmm, I take a detailed look on the impact for `_prepare_backbone_network()`, looks like `BACKBONE_IPV6_ADDR_START_BASE` is still needed by `_prepare_backbone_network()`. I will go what you have suggested. Thanks!